packages/react-catalog-view-extension/sass/react-catalog-view-extension/_catalog-tile.scss

```diff
@@ -22,22 +22,22 @@
   padding-bottom: 16px;
 
   .catalog-tile-pf-title {
-    font-size: ($font-size-base + 1);
+    font-size: 15px;
     font-weight: 400;
   }
 
   .catalog-tile-pf-subtitle {
-    color: $color-pf-black-500;
-    font-size: ($font-size-base - 1);
+    color: $pf-color-black-500;
+    font-size: 13px;
     font-weight: initial;
 
     a {
-      color: $color-pf-blue-300;
+      color: $pf-color-blue-300;
       text-decoration: none;
     }
 
     a:hover {
-      color: $color-pf-blue-200;
+      color: $pf-color-blue-200;
```

Comment:
[PF3 is `#7dc3e8`](https://github.com/patternfly/patternfly-3/blob/1d5ef31f3ccdae807ee9215b3f420d4243be1e64/src/sass/converted/patternfly/_color-variables.scss#L14)

[PF4 is `#73bcf7`](https://github.com/patternfly/patternfly/blob/9cc522b54163558ca0dfce4c35e2c80cd81cc2ab/src/patternfly/sass-utilities/colors.scss#L16)